src/ComplexType.php

```diff
@@ -6,11 +6,16 @@
 namespace Wsdl2PhpGenerator;
 
 use \Exception;
-use Wsdl2PhpGenerator\PhpSource\PhpClass;
-use Wsdl2PhpGenerator\PhpSource\PhpDocComment;
-use Wsdl2PhpGenerator\PhpSource\PhpDocElementFactory;
-use Wsdl2PhpGenerator\PhpSource\PhpFunction;
-use Wsdl2PhpGenerator\PhpSource\PhpVariable;
+use Wsdl2PhpGenerator\ZendCode\VarTag;
+use Zend\Code\Generator\ClassGenerator;
```

Comment:
I have looked though usages. In fact alias is needed only when there are some ```\Wsdl2PhpGenerator\ClassGenerator``` usages in file. But I will prefer to add alias in **all** places to eliminate a possibility of misunderstanding.

Another way can be class renaming, but I have no idea what new name should be.

What's your opinion about this?